docs/architecture/assets.md

```diff
@@ -1,22 +1,27 @@
-# Assets
 In Miden, users can create and trade arbitrary fungible and non-fungible assets.
 
 We differentiate between native and non-native assets in Miden. Native assets follow the Miden asset model. Non-native assets are all other data structures of value that can be exchanged.
 
-Recording of native assets in Polygon Miden suffices four goals:
+Native assets in Polygon Miden have four goals:
 
-* Asset exchange should be parallelizable
-* Asset ownership should be private
-* Asset usage should be indeed censorship resistant
-* Fees can be paid using any asset
+* Asset exchange should be parallelizable.
+* Asset ownership should be private.
+* Asset usage should be censorship resistant.
+* Fees can be paid using any asset.
 
-All native assets in Miden are stored directly in accounts, like Ether in Ethereum. Miden does not track ownership of assets using global hashmaps, e.g., ERC20 contracts. Storage of assets locally in accounts provides privacy and the ability for client-side proofs. That is because ownership changes always involve only one account and not the change of a global hashmap. Thus, they can happen in parallel. Additionally, asset exchange is censorship resistant at this level because there is no global contract the transfer must pass through. Finally, users can pay fees in any asset.
+All native assets in Miden are stored directly in accounts, like Ether in Ethereum. Miden does not track asset ownership using global hashmaps, e.g., ERC20 contracts. Local asset storage in accounts provides privacy and the ability for client-side proofs. That is because ownership changes are reflected only on an account and not on a global hashmap. Thus, these changes can happen in parallel. Additionally, asset exchange is censorship resistant at this level because there is no global contract the transfer must pass through. Finally, users can pay fees in any asset.
```

Comment:
I think this difference is important: 

`That is because ownership changes are reflected only on an account and not on a global hashmap.` 

We should emphasize that the ownership change in Miden is reflected only in the owning account and not an ERC20 account (which is a global hashmap).